optuna/terminator/improvement/evaluator.py

```diff
@@ -105,12 +93,28 @@ def _get_top_n(
         top_n_mask = values >= top_n_val
         return normalized_params[top_n_mask], values[top_n_mask]
 
+    @staticmethod
+    def _get_beta(n_params: int, n_trials: int, delta: float = 0.1) -> float:
+        # TODO(nabenabe0928): Check the original implementation to verify.
+        # Especially, |D| seems to be the domain size, but not the dimension based on Theorem 1.
+        beta = 2 * np.log(n_params * n_trials**2 * np.pi**2 / 6 / delta)
+
+        # The following div is according to the original paper: "We then further scale it down
+        # by a factor of 5 as defined in the experiments in
+        # `Srinivas et al. (2010) <https://dl.acm.org/doi/10.5555/3104322.3104451>`__"
+        beta /= 5
+
+        return beta
+
+    @staticmethod
     def _compute_standardized_regret_bound(
```

Comment:
I removed initial `_`, but remain it as a staticmethod. Since this method is tied to the paper on which `RegretBoundEvaluator` is based, it is natural to leave it as a staticmethod.